Directory.Packages.props

```diff
@@ -22,11 +23,12 @@
     <PackageVersion Include="Microsoft.OpenApi.Readers" Version="1.6.14" />
     <PackageVersion Include="Microsoft.NET.Test.Sdk" Version="17.9.0" />
     <PackageVersion Include="Newtonsoft.Json" Version="13.0.3" />
+    <PackageVersion Include="NSubstitute" Version="5.1.0" />
     <PackageVersion Include="Nswag.MSbuild" Version="13.17.0" />
     <PackageVersion Include="ReportGenerator" Version="5.2.2" />
     <PackageVersion Include="System.Text.Json" Version="4.6.0" />
     <PackageVersion Include="xunit" Version="2.7.0" />
     <PackageVersion Include="xunit.core" Version="2.7.0" />
     <PackageVersion Include="xunit.runner.visualstudio" Version="2.5.7" />
   </ItemGroup>
-</Project>
+</Project>
```

Comment:
If you editor respects our `.editorconfig` settings, it should keep this final line endings and not remove them.

Reply:
I expected Visual Studio 2022 to do that... need to see if there is maybe a setting I need to set still